wiki/Skinning/Compendium/en.md

```diff
@@ -6,278 +6,368 @@
 # Compendium
 Compendium of all completed osu! skins.
 
-- ![][o!s] means the skin contains osu!standard elements.
-- ![][o!c] means the skin contains osu!catch elements.
-- ![][o!t] means the skin contains osu!taiko elements.
-- ![][o!m] means the skin contains osu!mania elements.
+- ![][o!s]  - means the skin contains osu!standard elements.
```

Comment:
should be ` - `; too many spaces; same below

Reply:
I can change that later, but the german version has the same formatting iirc.